### PR TITLE
Advanced paging support

### PR DESCRIPTION
**Pull request by stenver**

### CODE REVIEW

**Comment by madis** [comment]
:+1:

File: apiary.apib [hunk 1]
```diff
@@ -864,6 +865,11 @@ Lists all the operators that the manager can manage and configure. The manager n
             Authorization: Token 2pFuPlTYkH7T4MylrwxApA
             Accept: application/vnd.salemove.v1+json
 
++ Parameters
+    + page (optional, number, `1`) ... The requested page if pagination is used.
+    + per_page (optional, number, `30`) ... The number of operators that should be returned with the request.
+    + order (optional, string, 'asc') ... Wether the collection should be returned in descending or ascending order. Operators are sorted by time of creation. Defaults to ascending order.
```
Comment (wifilinker) [inline apiary.apib]: Typo `Wether`

File: apiary.apib [hunk 2]
```diff
@@ -487,7 +487,8 @@ The collection is paginated and sorted by id (descending).
     + start_date (optional, iso8601 date, `2015-10-01T13:59:42.316+03:00`) ... The starting date range of the requested engagements.
     + end_date (optional, iso8601 date, `2015-10-01T13:59:42.316+03:00`) ... The ending date range of the requested engagements.
     + page (optional, number, `1`) ... The requested page if pagination is used.
-
+    + per_page (optional, number, `30`) ... The number of engagements that should be returned with the request.
+    + order (optional, string, 'asc') ... Wether the collection should be returned in descending or ascending order. Engagements are sorted by starting time. Defaulting to descending order.
```
Comment (wifilinker) [inline apiary.apib]: should be `Whether`

Comment (deiwin) [inline apiary.apib]: Inconsistency between `Defaulting to...` and `Defaults to...` below.

**Comment by stenver** [comment]
Updated

**Comment by deiwin** [comment]
!squash

**Pull request merged**